workflow/tests/base_results/results_bills.csv

```diff
@@ -0,0 +1,330 @@
+HPXML,Electricity: Fixed ($),Electricity: Marginal ($),Electricity: PV Credit ($),Electricity: Total ($),Natural Gas: Fixed ($),Natural Gas: Marginal ($),Natural Gas: Total ($),Fuel Oil: Total ($),Propane: Total ($),Wood Cord: Total ($),Wood Pellets: Total ($),Coal: Total ($),Total ($)
+base-appliances-coal.xml,144.0,1240.57,0.0,1384.57,96.0,120.73,216.73,0.0,0.0,0.0,0.0,72.99,1674.29
+base-appliances-dehumidifier-ief-portable.xml,144.0,1021.02,0.0,1165.02,96.0,7.41,103.41,0.0,0.0,0.0,0.0,0.0,1268.43
+base-appliances-dehumidifier-ief-whole-home.xml,144.0,1023.5,0.0,1167.5,96.0,7.15,103.15,0.0,0.0,0.0,0.0,0.0,1270.65
+base-appliances-dehumidifier-multiple.xml,144.0,1016.46,0.0,1160.46,96.0,7.93,103.93,0.0,0.0,0.0,0.0,0.0,1264.39
+base-appliances-dehumidifier.xml,144.0,1019.39,0.0,1163.39,96.0,7.2,103.2,0.0,0.0,0.0,0.0,0.0,1266.5900000000001
```

Comment:
It looks like the values in the individual results_bills.csv needs to be rounded to 2 decimals?